databases/permissions/src/main/conversions/v2.8.0/c280_2016072501.clj

```diff
@@ -0,0 +1,18 @@
+(ns facepalm.c280-2016072501
+  (:use [kameleon.sql-reader :only [load-sql-file]]))
+
+(def ^:private version
+  "The destination database version."
+  "2.8.0:20160725.01")
+
+(defn- add-version-table
+  "Adds a version table to the database."
+  []
+  (println "\t* adding a version table to the database.")
+  (load-sql-file "tables/version.sql"))
+
+(defn convert
+  "Performs the conversion for this database version."
+  []
+  (println "Performing the conversion for" version)
+  (add-version-table))
```

Comment:
I added this conversion because I was expecting to have to make other changes to the permissions database. The other changes turned out not to be necessary. This change is necessary, however, so I left it in.